flight/odlc_search_grid/plotter.py

```diff
@@ -0,0 +1,26 @@
+"""
+Provides plotting functionality for visualizing coordinate data
+"""
+
+from typing import List, Tuple
+import matplotlib.pyplot as plt
+
+
+def plot_data(search_paths: List[List[Tuple[float, float]]]) -> None:
+    """Simple plotter function to plot the search paths
+
+    Parameters
+    ----------
+    search_paths : List[Tuple[float, float]]
+        A list of search paths to be plotted
+    """
+
+    for path in search_paths:
+        x, y = [], []
```

Comment:
Don't forget your type hints; label what exactly these two lists are and what they contain, similar in format to the return type in your docstring.